src/main/2.7.0/org/apache/hadoop/hdfs/server/namenode/VersionContext.java

```diff
@@ -160,7 +160,7 @@ public void dumpINodeInDetail(String path, HttpServletResponse resp) throws IOEx
 
```

Comment:
This change needs to be replicated across the 2.4.0->2.9.0 shim directories and the 3.0.0 shim directory.